.github/workflows/check.yaml

```diff
@@ -30,3 +30,15 @@ jobs:
         go-version: ${{ env.GO_VERSION }}
     - name: Build cmd
       run: go build -o ./.out/nmpolicy ./cmd/nmpolicy
+  test:
+    name: test
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+    - name: Set up Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ env.GO_VERSION }}
+    - name: Run tests
+      run: test ./...
```

Comment:
`go test -v ./tests/...` for the integration tests and `go test -v ./nmpolicy/...` for the unit-tests.

Reply:
You are missing internal for the unit test
`go test -v ./nmpolicy/ ./internal`